pkg/operator/ceph/version/version.go

```diff
@@ -128,7 +132,13 @@ func ExtractCephVersion(src string) (*CephVersion, error) {
 		}
 	}
 
-	return &CephVersion{major, minor, extra, build}, nil
+	// get build suffix if running on a build release
+	mmm := buildSuffixPattern.FindStringSubmatch(src)
```

Comment:
nit: how about a more descriptive name than `mmm`? It's interesting to follow the pattern of the rest of the method, but I'd vote for descriptive names instead of a pattern like that. :)

Reply:
updated. Using `<>Match` as variable names